src/sidepanel/pages/Entry/index.tsx

```
@@ -1,26 +1,61 @@
+  const [track, setTrack] = useState<string>("");
+
+  useEventManager("ginkgoo-message", (message) => {
+    // console.log('🚀 ~ useEventManager ~ data:', message);
+
+    const { type: typeMsg, pilotInfo: pilotInfoMsg } = message || {};
+    const { caseId: caseIdMsg, workflowId: workflowIdMsg } = pilotInfoMsg || {};
+
+    switch (typeMsg) {
+      case "ginkgoo-background-all-polit-query-actived": {
+        const trackTmp = !!pilotInfoMsg
+          ? UtilsManager.router2url("/case-detail", {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)